frontend/next.config.js

```diff
@@ -4,7 +4,7 @@ const nextConfig = {
   images: {
     domains: [
       'localhost',
-      process.env.NEXT_PUBLIC_SERVER_URL.substring(8), // Remove https://
+     // process.env.NEXT_PUBLIC_SERVER_URL.substring(8), // Remove https://
```

Comment:
This is necessary to display images on the site. Please do not comment this out

Reply:
How do i run the project locally  if i comment that out?

Reply:
`NEXT_PUBLIC_SERVER_URL` can either be the CMS URL (aka `https://cms.manipalosf.org`) or the locally hosted CMS URL.
Without these being specified, all images served from the CMS will break